scripts/release.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Prepare for a release
+#
+# All additional options are passed to `rooster release`
+set -eu
+
+script_root="$(realpath "$(dirname "$0")")"
+project_root="$(dirname "$script_root")"
+
+echo "Updating metadata with rooster..."
+cd "$project_root"
+uv tool run --from rooster-blue --isolated -- rooster release "$@"
+
+echo "Updating lockfile..."
+cargo update -p ruff
+
+echo "Generating contributors list..."
+echo ""
+echo ""
+rooster contributors --quiet
```

Comment:
```suggestion
uv tool run --from rooster-blue --isolated -- rooster contributors --quiet
```

Reply:
Ah thanks — I dropped the contributors display over in `uv`.

Maybe we should merge this one after #9559

Reply:
Merged